### ISSUE TITLE
Click on overlay to close is not working on iPhone (Safari)

### ISSUE THREAD

**Issue by Mithrilhall** [tsubik/ember-side-menu#14]
Any suggestions on how to get this to work?

**Comment by Hypernikao** [comment]
Hey @Mithrilhall I had obliged to fix it by adding a "touchEvent" on content-backdrop.js (L.39) in node-modules:
```
click() {
        get(this, "sidceMenu").close();
  }

touchStart() {
        get(this, "sidceMenu").close();
  }
```
I know is a uggly solution :( but it works fine !

**Comment by Hypernikao** [comment]
Hey @Mithrilhall this happens because only click event is managed by the plugin.
One simple solution is to create a new component that inherits from ember-side-menu to add your custom behavior. Please consider the follow as an example of implementation:
```
> /components/custom-backdrop.js
import ContentBackdrop from 'ember-side-menu/components/content-backdrop';

export default ContentBackdrop.extend({

  touchStart() {
    this.get('sideMenu').close();
  }
});
```
Hope it will help you :)

**Comment by Mithrilhall** [comment]
Thanks @Hypernikao ... I'll give that a shot and I'll reopen if it doesn't work.

**Comment by tsubik** [comment]
Thanks @Hypernikao and @Mithrilhall . Fixed in https://github.com/tsubik/ember-side-menu/commit/af1f19aa3af5cc1903fc62549ab26ff5c04b85c1 and new published version 0.0.15.
